Scripts/script-DeleteConext.yml

```
@@ -53,7 +53,7 @@ script: |
           ReadableContentsFormat: formats.markdown,
           EntryContext: keysToKeepObj
       };
-  } else if (args.index) {
+  } else if (args.index != undefined)) {
```

Best to use strict compare
```suggestion
  } else if (args.index !== undefined)) {
```

---

Never mind. For undefined strict compare is not needed.